dashboard/lib/services/script_seed.rb

```diff
@@ -185,6 +185,7 @@ def self.seed_from_json(json_string)
 
     def self.import_script(script_data)
       script_to_import = Script.new(script_data.except('seeding_key'))
+      script_to_import.is_migrated = true
```

Comment:
just noting here that this line means that running the new seed script will attempt to mark the new script as migrated, which, if it is visible (any live CSF/CSD/CSP/HOC course) should fail. I verified this on one of the adhocs:
![Screen Shot 2020-12-18 at 9 16 20 AM](https://user-images.githubusercontent.com/8001765/102641919-be6d0f80-4111-11eb-83a0-a315fc2adfab.png)

Reply:
![Screen Shot 2020-12-18 at 9 17 09 AM](https://user-images.githubusercontent.com/8001765/102641987-d6449380-4111-11eb-9c76-ea6461af73ba.png)